src/file-handler.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
+
+/**
+ * @return {String} The content of the read file.
+ * @param {String} filePath The path to the file to be read.
+ */
+
+
+export default {
+  getContent(filePath, callback) {
+    fs.readFile(filePath, 'utf8', (err, data) => {
+      // fs.unlink(filePath);  // Delete  file after reading to save space
```

Comment:
Usually, during testing, jasmine raises permission issues when deleting files after a certain number of runs, except I restart. So, to have a smooth testing, I turn off that feature before running tests. I usually un-comment it before a push. That was a gross oversight  this time. I'll fix that.

PS: This permission issue doesn't arise during a travis build, so it doesn't affect the functionality of my app. Running git as administrator sometimes fixes the issue for me.